opsdroid/loader.py

```diff
@@ -536,7 +536,7 @@ def _install_module(self, config):
         Args:
             self: instance method
             config: dict of module config fields
-
+    
```

Comment:
Linting is failing due to the white space on this blank line. Could you remove it please?